src/js/tabs/submit.js

```diff
@@ -19,9 +19,11 @@ SubmitTab.prototype.generateHtml = function ()
 
 SubmitTab.prototype.angular = function (module)
 {
-  module.controller('SubmitCtrl', ['$scope', '$location', 'rpFileDialog', 'rpNW',
-    function ($scope, $location, fileDialog, rpNW)
+  module.controller('SubmitCtrl', ['$scope', '$location', 'rpFileDialog', 'rpNW', 'rpNetwork',
+    function ($scope, $location, fileDialog, rpNW, $net)
     {
+      $net.connect();
```

Comment:
I was thinking it could just connect to the network when the page initially loads. Does it matter if it's inside TxRowCtrl?